doc/manual/src/command-ref/nix-store.md

```diff
@@ -465,7 +465,7 @@ Make a picture of the runtime dependency graph of the current user
 environment:
 
 ```console
-$ nix-store -q --graph ~/.nix-profile | dot -Tps > graph.ps
+$ nix-store -q --graph $XDG_DATA_HOME/nix/profile | dot -Tps > graph.ps
```

Comment:
Examples shouldn't use `XDG_DATA_HOME` because it may not be set (it isn't on my system). It's better to just use `~/.local/share/nix/profile` in examples since very few people will use something else.

Maybe the same applies to the documentation, since references to `$XDG_DATA_HOME/nix/profile` are harder to understand than `~/.local/share/nix/profile`.

Reply:
I fully agree for this example here, but for documentation I think it's important to let the people know we're (kinda) following the XDG spec and not just putting things in `~/.local/share`.